pxd.h

```diff
@@ -146,11 +147,12 @@ struct pxd_read_data_out {
 };
 
 /**
- * PXD_UPDATE_SIZE request from user space
+ * PXD_UPDATE_SIZE ioctl from user space
  */
-struct pxd_update_size_out {
+struct pxd_update_size {
```

Comment:
have you made sure all references to pxd_update_size_out are taken care? Looks like pxdriver still has it.